.travis.yml

```diff
@@ -52,18 +52,18 @@ install:
       curl -O -L https://www.libsdl.org/projects/SDL_image/release/SDL2_image-2.0.1.dmg;
       curl -O -L https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-2.0.1.dmg;
       curl -O -L https://www.libsdl.org/projects/SDL_ttf/release/SDL2_ttf-2.0.13.dmg;
-      curl -O -L http://gstreamer.freedesktop.org/data/pkg/osx/1.6.2/gstreamer-1.0-1.6.2-x86_64.pkg;
-      curl -O -L http://gstreamer.freedesktop.org/data/pkg/osx/1.6.2/gstreamer-1.0-devel-1.6.2-x86_64.pkg;
+      curl -O -L https://www.googledrive.com/host/0B1_HB9J8mZepS0hXamNnd2pRWkE/gstreamer-1.0-1.6.2-x86_64.pkg;
```

Comment:
The reason for this, with the old links directly from libsdl their connection was so slow, that it'd take at least half an hour to download and fairly often the tests would time out. However, from google drive it takes a few seconds to download making the test result a lot quicker.

Should I still revert? What is the problem with hosting on drive?